docs/user-guide.md

```diff
@@ -28,8 +28,8 @@ published: True
       - [NFS share](#nfs-share)
     - [Validating Media](#validating-media)
   - [Prerequisite configuration](#prerequisite-configuration)
-    - [Data mount configuration file](#data-mount-configuration-file)
-    - [ASM disk group configuration file](#asm-disk-group-configuration-file)
+    - [Data mount configuration](#data-mount-configuration)
+    - [ASM disk groups configuration](#asm-disk-groups-configuration)
```

Comment:
SOrry to be a pain here, but the same comments for data mounts apply to ASM disk groups (as you can have just one):  ASM disk group configuration